config/consensus.go

```diff
@@ -1160,6 +1163,7 @@ func initConsensusProtocols() {
 	vFuture.EnableSHA256TxnCommitmentHeader = true
 
 	// Boxes (unlimited global storage)
+	vFuture.MaxBoxSize = 8096
```

Comment:
how temporary is this value?

Reply:
I just put it in so we have the code written to use it, and it then becomes obvious we have to choose the value.

There is still _another_ limit that is probably what we really need to pin down: the total size of the boxes specified in the static arrays of a txgroup.